content/md/what-is/what-is-an-asset.md

```diff
@@ -0,0 +1,53 @@
+---
+id: what-is-an-asset
+themes: first-steps, akeneo-concepts, manage-your-assets
+title: What is an **asset**?
+popular: false
+ee-only: true
+related: assets-transformation, upload-assets, work-with-assets
+---
+
+# Definition of an asset
+
+An asset can be an HD image, a video, a PDF installation guide... Any kind of file that can be used to enrich a product in Akeneo.
+
+::: ee
+Assets are only available in the Enterprise Edition.
+:::
+
+An asset is always `scopable` because you made need asset reference file variations for each channel - for example, a full size image for print and a small image for ecommerce.  
```

Comment:
An asset is always `scopable` because you *may* need asset reference file variations for each channel

Reply:
@jmevans00 What do you think? Is "made" a typo or what?

Reply:
```suggestion
An asset is always `scopable` because you may need asset reference file variations for each channel - for example, a full size image for print and a small image for ecommerce.  
```